doc/build/dts/intro.rst

```diff
@@ -166,24 +162,20 @@ This is the corresponding DTS:
 
    / {
            soc {
-                   i2c@40003000 {
+                   i2c_0:i2c@40003000 {
```

Comment:
Do you really need the node labels here? It looks like they aren't used. They complicate the example, so since they are not required and users have already been told that zero is a legal number of node labels, I don't think adding node labels here makes the documentation clearer.